frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -3,20 +3,53 @@ import './LandingPage.css'
 const LandingPage = () => {
     return (
         <div className="home-body-div">
-            <div className="home-top-div">
-                <div className="logo-div">
-                    <img src='/logo-dark.png' className='dark-logo' />
+            <div className="top-bottom-div">
+
+                <div className="home-top-div">
+                    <div className="logo-div">
+                        <img src='/logo-dark.png' className='dark-logo' />
```

Comment:
The source here will need to be changed to `static/logo-dark.png` — this is currently broken in production.

Reply:
Oh, and the `logo-dark.png` will also have to be moved into `static/`